src/components/QuickOpenModal.js

```diff
@@ -68,10 +70,18 @@ type GotoLocationType = {
 };
 
 function filter(values, query) {
-  return fuzzyAldrin.filter(values, query, {
+  let results = fuzzyAldrin.filter(values, query, {
     key: "value",
     maxResults: 1000
   });
+
+  if (prefs.projectDirectoryRoot) {
+    results = results.filter(result =>
```

Comment:
I feel like we should have a selector for getting the filtered sources... Not sure if getSources should do it, but that could be a next step based on the issue in discussion